.github/workflows/ci.yml

```diff
@@ -223,6 +224,7 @@ jobs:
         PYTHON:
           - {VERSION: "3.7", NOXSESSION: "tests-nocoverage"}
           - {VERSION: "3.11", NOXSESSION: "tests"}
+          - {VERSION: "3.12", NOXSESSION: "tests"}
```

Comment:
On macOS and Windows, we test on only two versions of Python: our minimum, and our maximum.